### ISSUE TITLE
Support alertmanager as alert source

### ISSUE THREAD

**Issue by freeseacher** [StackStorm-Exchange/stackstorm-prometheus#7]
Don't know if there are good place to add this issue but anywhere. 

Since https://github.com/prometheus/alertmanager/pull/1588 
There can be cough raw alerts so direct integration is probably unneeded. It is good point to add support alertmanager as supported event source.

**Comment by nzlosh** [comment]
+1 for this feature.

Alert Manager events are aggregated which makes handling them via a generic StackStorm webhook, at best, clumsy or worse impossible to process using the IFTTT rule engine.

A sensor needs to be developed that accepts Alert Manager emitted events and generates discrete StackStorm trigger instances.  This would allow plain and simple IFTTT rules to be used to direct various alerts to the appropriate action to handle them.

**Comment by nmaludy** [comment]
@nzlosh yeah we can either do a polling sensor or a sensor based on a webhook and configure AlertManager to send webhooks to StackStorm
